compiler/docgen.nim

```diff
@@ -614,6 +614,25 @@ type RunnableState = enum
   rsRunnable
   rsDone
 
+when false:
+  proc toStr(a: ItemFragment): string =
```

Comment:
useful for debugging; I've used this in a few PR's; i can remove or move to another PR if needed